devdoc/constbuffer_array_requirements.md

```diff
@@ -54,6 +55,24 @@ MOCKABLE_FUNCTION(, CONSTBUFFER_ARRAY_HANDLE, constbuffer_array_create, const CO
 
 **SRS_CONSTBUFFER_ARRAY_01_014: [** If any error occurs, `constbuffer_array_create` shall fail and return NULL. **]**
 
+### constbuffer_array_create_with_move_memory
+
+```c
+MOCKABLE_FUNCTION(, CONSTBUFFER_ARRAY_HANDLE, constbuffer_array_create_with_move_memory, CONSTBUFFER_HANDLE*, buffers, uint32_t, buffer_count);
```

Comment:
>memory [](start = 81, length = 6)

poetry: given the fact that what is moved is "buffers", can we call this API what it is? suggestion: constbuffer_array_create_with_move_buffers :)

Reply:
Renamed.

---
In reply to: [299235405](https://github.com/Azure/azure-c-shared-utility/pull/338#discussion_r299235405) [](ancestors = 299235405)